### ISSUE TITLE
Allow to style the border of an element independently

### ISSUE THREAD

**Issue by ilmoralito** [craigburke/document-builder#27]
I would like to style some element border individually in a way where i could set border-top to 0.inch border-right to 1.inch.

**Comment by ilmoralito** [comment]
I would like to style some element border individually i a way where i could set border-top 0.inch border-right 1.inch.

**Comment by goeh** [comment]
Agree. In my current project I need to set border-top only. I don't want left, right or bottom border on my table, only top. There must be a way to add and style borders individually. Setting width to 0 could be a viable solution.
I'm skipping this requirement for now but will get back to this issue later to see if I can help.
